Strategies/protocols/psyfi/actions/deposit.ts

```diff
@@ -101,11 +163,28 @@ export const deposit = async (
       transferAddress,
       vaultOwnershipAccount
     )
+    additionalTokenInstructions.push(
+      serializeInstructionToBase64(tokenInstruction)
+    )
+  }
+
+  if (treasuryAssetAccount.isSol) {
```

Comment:
i think we can skip this - the treasury account will just end up with an wSOL associated token account.

Reply:
We can but the transfer instructions should still be only executed when its native SOL right?